cypress/support/commands.js

```diff
@@ -250,8 +248,102 @@ Cypress.Commands.add('checkElementComponentContainsValue', (mainSelector, compon
  * @param {Array} values Array of string values
  */
 
- Cypress.Commands.add('checkValuesExistInComponent', (selector, value) => {
-    cy.wrap(value).each((str) => {
-      cy.get(selector).contains(new RegExp('^' + str + '$'))
+Cypress.Commands.add('checkValuesExistInComponent', (selector, value) => {
+  cy.wrap(value).each((str) => {
+    cy.get(selector).contains(new RegExp('^' + str + '$'))
+  })
+})
+
+/**
+ * Read indices from a file and send to OpenSearch using the create index API
+ * @param {String} filename File path (with its root at the directory containing the cypress.json file)
+ * @param {String} hostname Host name for OpenSearch
+ * @param {String} port Port for OpenSearch
+ */
+
+Cypress.Commands.add('importJSONMapping', (filepath, hostname = 'localhost', port = '9200') => {
```

Comment:
Nonblocking: we can later refactor these loading/unloading commands to their own file, say data_commands.js or json_commands.js

Reply:
I agree, the next PR can address splitting the `commands.js` file into multiple, more descriptive files.